pom.xml

```diff
@@ -20,7 +20,7 @@
 		<git.revision>latest</git.revision>
 		<zkclient.version>0.11</zkclient.version>
 		<kafka-clients.version>2.4.0</kafka-clients.version>
-		<node.version>v12.13.1</node.version>
+		<node.version>v14.15.4</node.version>
```

Comment:
I'm not sure we need to update pom files manually. @soffest could you please take a look?